xarray/core/formatting.py

```diff
@@ -272,7 +272,8 @@ def summarize_datavar(name, var, col_width):
 
 
 def summarize_coord(name, var, col_width):
-    is_index = name in var.dims
+    from .variable import IndexVariable
```

Comment:
I tried moving this import to the top of `formatting.py` but, for reasons I can't understand, I got this error
```
    from .variable import IndexVariable
E   ImportError: cannot import name 'IndexVariable
```

Why does it work in the function but not at the top?

Reply:
I think there is a circular import going on here, as .variable does import .common which does import .formatting Perhaps adding a self.is_index attribute to the Variable class could be a way to avoid this import?

Reply:
Yes, we have a circular import here. We could potentially refactor to avoid but I think this is also an OK work-around.